internal/helper/yawollet.go

```diff
@@ -344,17 +356,25 @@ func createEnvoyTCPListener(
 			},
 		},
 		FilterChains: []*envoylistener.FilterChain{{
-			// proxy filter has to be the last in the chain
-			Filters: append(filters, &envoylistener.Filter{
-				Name: envoywellknown.TCPProxy,
-				ConfigType: &envoylistener.Filter_TypedConfig{
-					TypedConfig: listenPort,
+			Filters: append(filters,
+				&envoylistener.Filter{
```

Comment:
Is there an ordering constraint like "must/should be first in chain?" for this filter? Looks like you want to drop those connections fast before filtering further.

Reply:
```

Reply:
I also thought about that but the only filter before is the rbac filter which is kind of a "security" filter. To me it seems like this one should always be the first in the chain.

What do you think about that? I can still put the filter into its own statement after the if statement, so it is easier to decide for new filters at which position they should go later on.